api/WindowControlsOverlay.json

```diff
@@ -0,0 +1,201 @@
+{
+  "api": {
+    "WindowControlsOverlay": {
+      "__compat": {
+        "mdn_url": "https://developer.mozilla.org/docs/Web/API/WindowControlsOverlay",
+        "spec_url": "https://wicg.github.io/window-controls-overlay/#windowcontrolsoverlay-interface",
+        "support": {
+          "chrome": {
+            "version_added": "98"
+          },
+          "chrome_android": {
+            "version_added": false
+          },
+          "edge": {
+            "version_added": "98"
+          },
+          "firefox": {
+            "version_added": false
+          },
+          "firefox_android": {
+            "version_added": false
+          },
+          "ie": {
+            "version_added": false
+          },
+          "opera": {
+            "version_added": "84"
+          },
+          "opera_android": {
+            "version_added": false
+          },
+          "safari": {
+            "version_added": false
+          },
+          "safari_ios": {
+            "version_added": false
+          },
+          "samsunginternet_android": {
+            "version_added": false
+          },
+          "webview_android": {
+            "version_added": false
+          }
+        },
+        "status": {
+          "experimental": true,
+          "standard_track": true,
+          "deprecated": false
+        }
+      },
+      "geometrychange_event": {
```

Comment:
Is this right? The linked spec seems to say the event is "change" (implying `change_event` here). I haven't actually used the API though, so it's possible the actual implementation is different

Reply:
Yeah, there was the same confusion with the event name on the MDN PR. But testing live in chromium shows that the event name is geometrychange. So I think we should keep it like it is here.

Reply:
Agreed.